test/integration-http-server.cpp

```diff
@@ -168,7 +169,35 @@ void HTTPServerTest::testCoolPostPoco()
     Poco::StreamCopier::copyToString(rs, html);
 
     LOK_ASSERT(html.find(form["access_token"]) != std::string::npos);
+    LOK_ASSERT(html.find(form["buy_product"]) != std::string::npos);
     LOK_ASSERT(html.find(_uri.getHost()) != std::string::npos);
+
+    std::string csp = response["Content-Security-Policy"];
+    StringVector lines = StringVector::tokenize(csp, ';');
+    LOG_TST("CSP - " << csp << " tokens " << lines.size());
+    for (size_t i = 0; i < lines.size(); ++i)
+    {
+        if(lines.startsWith(i, " connect-src") ||
```

Comment:
Are these necessarily to start with a space? Perhaps we should allow for no spaces?

Reply:
If C++ had a nice 'chomp' function that wasn't an explosion of evil - I'd use it; but as-is, this works fine for the test =)